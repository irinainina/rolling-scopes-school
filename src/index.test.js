import render from './index';

describe('render is a function for render HTML', () => {
  it('Should be an instance of a function', () => {
    expect(render).toBeInstanceOf(Function);
  });

  it('Should be render correctly', () => {
    const context = {
      titles: [
        'Video about JS',
      ],
    };
    render.call(context);
    expect(document.body.innerHTML).toMatchSnapshot();
  });
});
