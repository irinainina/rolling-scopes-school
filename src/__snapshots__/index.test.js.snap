// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render Should be render correctly 1`] = `"<div class=\\"search\\"><input autofocus=\\"\\" placeholder=\\"Awesome Javascript\\"><button></button></div><div class=\\"container\\"><div class=\\"items\\"></div></div><div class=\\"pagination\\"><div class=\\"colorpicker\\"></div></div>"`;

exports[`render is a function for render HTML Should be render correctly 1`] = `"<div class=\\"search\\"><input autofocus=\\"\\" placeholder=\\"Awesome Javascript\\"><button></button></div><div class=\\"container\\"><div class=\\"items\\"></div></div><div class=\\"pagination\\"><div class=\\"colorpicker\\"></div></div>"`;
